Library/Formula/kafka.rb

```diff
@@ -42,19 +41,21 @@ def install
     inreplace "config/zookeeper.properties",
       "dataDir=/tmp/zookeeper", "dataDir=#{data}/zookeeper"
 
-    # Workaround for conflicting slf4j-log4j12 jars (1.7.6 is preferred)
-    rm_f "core/build/dependant-libs-2.10.4/slf4j-log4j12-1.6.1.jar"
+    # Workaround for conflicting slf4j-log4j12 jars (1.7.10 is preferred)
+    rm_f "core/build/dependant-libs-2.10.4/slf4j-log4j12-1.7.6.jar"
 
     # remove Windows scripts
     rm_rf "bin/windows"
 
-    libexec.install %w[clients contrib core examples system_test]
+    libexec.install %w[clients core examples]
 
     prefix.install "bin"
     bin.env_script_all_files(libexec/"bin", Language::Java.java_home_env("1.7+"))
 
     mv "config", "kafka"
-    etc.install "kafka"
+    unless File.exists? etc+"kafka"
```

Comment:
You don't need to do this, it's handled already by `etc.install` which doesn't overwrite files.